src/components/dashboard/viewDriver.js

```diff
@@ -5,7 +5,7 @@ import  '../../styles/dashboard/driverdashboard.scss'
 export const DriverComponent = ({handleSubmit, handleChange,obj})=>(
 
 
-            <Card>
+            <Card className={'mycard'}>
```

Comment:
would you make this classname more specific?

Reply:
instead of calling it my `mycard`, you may name it something specific to the content it is working on